python/README.md

```diff
@@ -1,8 +1,6 @@
-# Foxglove Studio WebSocket server
+# Foxglove WebSocket server
 
-This package provides a server implementation of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol) supports any data encoding, as long as it is supported by both client and server. This package provides an example server implementation of the protocol, enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
```

Comment:
Seems like the order of sentences is basically being swapped here. I find it more intuitive the original way: starting with "what is this package for", and then a second sentence which gives some details on the nature of the protocol. We could also consider just removing the "encoding-agnostic" sentence, since it's arguably a kind of low-level detail, which people can already learn by clicking on the link to the main docs page.